providers/redis/shedlock-provider-redis-quarkus2/src/main/java/net/javacrumbs/shedlock/provider/redis/quarkus2/SchedulerLockFactory.java

```diff
@@ -0,0 +1,25 @@
+package net.javacrumbs.shedlock.provider.redis.quarkus2;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import io.quarkus.redis.datasource.RedisDataSource;
+import net.javacrumbs.shedlock.core.LockProvider;
+
+
+@ApplicationScoped
+public class SchedulerLockFactory {
```

Comment:
Do we need this? In Spring we let the users to specify the Lock Provider in the aplication code. Having it in the library feels surprising.

Reply:
I think if I import a specific provider it's because I want to use it... I don't see the need to configure it manually